var/spack/repos/builtin/packages/actsvg/package.py

```diff
@@ -18,6 +18,7 @@ class Actsvg(CMakePackage):
 
     maintainers("HadrienG2", "wdconinc")
 
+    version("0.4.30", sha256="f7ffea39b3132914fcbb0fac6ab7395bef295cd6078dfd1c2509fd2d9aab0acb")
```

Comment:
We could add 0.4.29 for completeness (I've been using it locally, but hadn't pushed a PR yet):
```suggestion
    version("0.4.30", sha256="f7ffea39b3132914fcbb0fac6ab7395bef295cd6078dfd1c2509fd2d9aab0acb")
    version("0.4.29", sha256="971f4f344c3143b654e6a86422534c6916f785f2c2c3785664c4ae7ddf2f5e4b")
```

Reply:
Good idea! Thanks for the suggestion!